docs/modules/fault-tolerance/pages/backups.adoc

```diff
@@ -7,6 +7,17 @@ The distribution happens on the partition level; the data and its backups are st
 memory partitions. See the xref:overview:data-partitioning.adoc[Data Partitioning] and xref:clusters:partition-group-configuration.adoc[Partition Grouping]
 for more information about the partitioning.
 
+[IMPORTANT]
+====
+Hazelcast does not support backup redistribution on data structure level. For example, for a map with a nonzero backup count,
+
+1. increasing the backup count does **not** create additional copies of existing map entries,
+2. decreasing the backup count does **not** remove any backups of existing map entries, so some backups will have stale data if the map entries are overwritten, and
+3. if some backups are lost due to node failure, additional copies are **not** created on other nodes to meet the backup count.
+
+Thus, it is unsupported to change the backup count of a data structure once it was populated with data.
```

Comment:
>it is unsupported to change the backup count of a data structure once it was populated with data

this should be the first sentence of the note. It does not have to be a mathematical proof and this is the most important info for the user, the rest is just a more detailed explanation.

Reply:
also, does it make a difference if the structure is empty (eg. after `clear`), destroyed or never contained any data?

Reply:
The limitation doesn't apply to nonempty data structures that once contained data. (ca51ef8c)